net.meijn.onvifviewer.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id": "net.meijn.onvifviewer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "onvifviewer",
+    "finish-args": [
+        "--socket=x11",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "kdsoap",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/caspermeijn/KDSoap.git",
+                    "branch": "master",
+                    "commit": "5fadb5eb393ed8aa234290a2001dbe92915907fb"
+                }
+            ]
+        },
+        {
+            "name": "onvifviewer",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.com/caspermeijn/onvifviewer.git",
+                    "tag": "v0.5",
+                    "commit": "07064ee340904d65fd5e5eb3c25b73c8fea28389"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/KDE/breeze-icons.git",
```

Comment:
breeze-icons is part of the runtime at the moment, no need to include again.

Reply:
Currently the breeze-icons are required for the application to build, as the icons are compiled into the binary for when a theme is not available. I will make this optional in a future version, so that this dependency can be removed.